packages/manager/src/components/PaymentMethodRow/PaymentMethodRow.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { CardType } from '@linode/api-v4';
+import React from 'react';
+
+import { paymentMethodFactory } from 'src/factories';
+
+import { PaymentMethodRow } from './PaymentMethodRow';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof PaymentMethodRow>;
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const onDelete = () => {};
```

Comment:
Should we use a Storybook action here? 


```suggestion
const onDelete = action('onDelete');
```